docset/winserver2022-ps/appx/Add-AppSharedPackageContainer.md

```diff
@@ -18,26 +19,29 @@ Add-AppSharedPackageContainer [-Path] <String> [-ForceApplicationShutdown] [-Mer
 ```
 
 ## DESCRIPTION
-The Add-AppSharedPackageContainer cmdlet deploys the shared package container definiton for the particular user.
+
+The Add-AppSharedPackageContainer cmdlet deploys the shared package container definition for the
```

Comment:
We want to make sure cmdlet names are wrapped in backticks, per the [style guide][01], so they render correctly and reliably.

```suggestion
The `Add-AppSharedPackageContainer` cmdlet deploys the shared package container definition for the
```

[01]: https://learn.microsoft.com/en-us/powershell/scripting/community/contributing/powershell-style-guide?view=powershell-7.3